portfolio/src/main/webapp/blog.html

```diff
@@ -7,8 +7,77 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
-  <body>
-    <h1 class="heading">STEP Blog</h1>
+  <body class="blog">
+    <h1 class="heading">STEP Internship Blog</h1>
     <div id="top-navigation"></div>
+    <br>
+    <p class="blog-intro">Welcome to my STEP blog! I will be updating this blog throughout my internship with all the 
+      fun and awesome events and learning experiences that I will hopefully have. Feel free to use the dropdowns 
+      below to view my thoughts on each week of my STEP internship!</p>
+    <br>
+    <div class="accordion-both accordion-left">
+      <button class="accordion">Week 1</button>
+      <div class="panel">
+        <p>During the first week of my internship, I went through onboarding and technical trainings 
+          and got to meet the team! I also got the opportunity to explore Google and all the amazing
+          ERGs and intern groups that were created to make our internship experience the best it can be. 
+          Although getting an influx of emails and go/ links was a bit overwhelminig, being able 
+          to take out time to explore everything made the first week very fun!
+        </p>
+      </div>
```

Comment:
This is probably fine but I wonder if there's a way to reuse this (maybe via a JS function). Since it looks like each section is repeated multiple times and the only differences are the week number and content text. But I'm not too familiar with HTML/JS. Maybe Linh would know?

Reply:
Yeah, I've been able to figure out how to reuse the same code for the navbar but not for "similar" code with different contents. I will definitely try to spend some time next week to see if there is a solution for this